### ISSUE TITLE
Errors with the new Stylify version

### ISSUE THREAD

**Issue by vmatiyko** [micmro/Stylify-Me#26]
Hi!

It seems the new Stylify version (with the Puppeteer lib) doesn't process some websites properly. There are some examples:
- { "code": "404", "msg": "Fail to load the current url - please make sure you don't have typos" }
	- https://ignitine.myshopify.com
	- https://shopforeverglammed.com
	- https://www.lemonwaterwellness.ca
	- https://ihouzit.com
	- https://lkjewellery.nl
- { "code": "503", "msg": "Sorry, our server experiences a high load and the service is currently unavailable" }
	- https://houseofglam.ca
	- https://www.indvcollective.com
	- https://mcmbeauty.com 
	- https://glam-lab.com 
	- https://www.makeupamurder.com
- { "code": "502", "msg": "Fail to parse site - the site might try to redirect or has invalid markup", "thumbPath": "temp-img/website_name__16637782112618609.png" }
	- https://www.naturesguru.com 
	- https://majestikhaircare.com

Also, http://stylifyme.com/ responds with the error:

```
Could not query site, our service might be down, please try again later.
```

I didn't dig into the puppeteer errors, but these websites were processed with the Phantomjs lib properly. Maybe I should tune the Puppeteer lib somehow?

**Comment by brunoalex** [comment]
i dont think this app is working , its missing some js in the public directory
